kie-remote/src/main/java/org/kie/remote/RemoteStreamingKieSession.java

```diff
@@ -31,4 +32,8 @@ static RemoteStreamingKieSession create( Properties configuration) {
     static RemoteStreamingKieSession create(Properties configuration, TopicsConfig envConfig) {
         return new RemoteStreamingKieSessionImpl( configuration, envConfig );
     }
+
+    CompletableFuture<String> getKJarGAV();
+
+    void updateKJarGAV(String kjar);
```

Comment:
Please create a `UpdatableSession` interface with these two methods and make both `RemoteStreamingKieSession` and `RemoteKieSession` extends it

Reply:
Ok